web3.js/examples/get_account_info.js

```diff
@@ -0,0 +1,23 @@
+import * as web3 from '@solana/web3.js';
+
+(async () => {
+  // Connect to cluster
+  var connection = new web3.Connection(
+    web3.clusterApiUrl('devnet'),
+    'confirmed',
+  );
+
+  // Generate a new wallet keypair and airdrop SOL
+  var wallet = web3.Keypair.generate();
+  var airdropSignature = await connection.requestAirdrop(
+    wallet.publicKey,
+    web3.LAMPORTS_PER_SOL,
+  );
+
+  //wait for airdrop confirmation
+  await connection.confirmTransaction(airdropSignature);
+
+  //get account info
+  let account = await connection.getAccountInfo(wallet.publicKey);
+  console.log(account);
```

Comment:
Could you also add a comment that says that `account.data` is a raw byte buffer and devs need to deserialize it themselves?

Reply:
Can we elaborate that a bit further to express that the serialization is program specific?